### PR TITLE
Added discord bots category

### PR DESCRIPTION
**Pull request by oinkrr**

### CODE REVIEW

**Review by jamesbt365 — APPROVED** [review]

File: content.md [hunk 1]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show of your profile/recent plays, get map recommendations and compare yourself to others
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich discord bot with functionality all around osu!
```
Comment (Ruben172) [inline content.md:92]: nowadays bathbot is a lot better, more feature-rich and kept more up-to-date, so I think it should be above owo, should also have a description closer to what owo has

File: content.md [hunk 2]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
...
+- [Yuna](https://discord.com/oauth2/authorize?client_id=832597585923014676&scope=bot&permissions=60480) - The official [o!rdr](https://ordr.issou.best/) bot lets you render your replays through discord chat
+- [AxerBot](https://github.com/Hiviexd/AxerBot) - A general-purpose and powerful Discord bot with osu! features related to mapping & modding
```
Comment (Ruben172) [inline content.md:94]: please change all instances of "discord bot" throughout these resources with "bot", it's already in the "Discord bots" category so rather redundant

File: content.md [hunk 3]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show of your profile/recent plays, get map recommendations and compare yourself to others
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich discord bot with functionality all around osu!
+- [Yuna](https://discord.com/oauth2/authorize?client_id=832597585923014676&scope=bot&permissions=60480) - The official [o!rdr](https://ordr.issou.best/) bot lets you render your replays through discord chat
```
Comment (Ruben172) [inline content.md:93]:
```suggestion
- [Yuna](https://discord.com/oauth2/authorize?client_id=832597585923014676&scope=bot&permissions=60480) - The official [o!rdr](https://ordr.issou.best/) bot, useful to render replays through discord chat
```

File: content.md [hunk 4]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show of your profile/recent plays, get map recommendations and compare yourself to others
```
Comment (Ruben172) [inline content.md:91]:
```suggestion
- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show off your profile/recent plays, get map recommendations and compare yourself to others
```

File: content.md [hunk 5]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
...
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich discord bot with functionality all around osu!
+- [Yuna](https://discord.com/oauth2/authorize?client_id=832597585923014676&scope=bot&permissions=60480) - The official [o!rdr](https://ordr.issou.best/) bot lets you render your replays through discord chat
+- [AxerBot](https://github.com/Hiviexd/AxerBot) - A general-purpose and powerful Discord bot with osu! features related to mapping & modding
+- [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - A program to analyze misses in an osu! replay (can be used locally or through discord)
```
Comment (Ruben172) [inline content.md:95]:
```suggestion
- [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - A tool to analyze misses in an osu! replay (has a discord bot and program to run locally)
```

kinda having second thoughts on moving this and thinking we might just want to put it in both instead, with links pointing to either the main/bot readme

**Review by Ruben172 — CHANGES_REQUESTED** [review]

File: content.md (hunk 1 above)
Comment (jamesbt365) [reply content.md:92]: Right now the resources in general in each category aren't exactly ordered, but considering reading order, yes.

Comment (Ruben172) [reply content.md:92]: yeah because I haven't bothered doing an in-depth look over the whole document yet lol

**Review by jamesbt365 — APPROVED** [review]

File: content.md [hunk 6]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich bot with functionality all around osu! - check your recent plays, compare top scores among players, play the bg guessing game
```
Comment (Ruben172) [inline content.md:91]:
```suggestion
- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich bot with functionality all around osu! - check your recent plays, compare top scores among players, play the background guessing game and more
```

File: content.md [hunk 7]
```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
...
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show off your profile/recent plays, get map recommendations and compare yourself to others
```
Comment (Ruben172) [inline content.md:92]:
```suggestion
- [owo](http://owo-bot.xyz/) - A general-purpose bot to show off your profile/recent plays, get map recommendations, compare yourself to others and more
```

**Review by Ruben172 — CHANGES_REQUESTED** [review]

**Review by Ruben172 — APPROVED** [review]

**Review by jamesbt365 — APPROVED** [review]

**Comment by jamesbt365** [comment]
Unrelated CI failure, will merge anyway.

**Pull request merged**